### PR TITLE
doc / update algo rewards value

### PR DESCRIPTION
**Pull request by PtrckM**
- update rewards value for algorand campaign

image

### CODE REVIEW

**Comment by thomasyit** [comment]
Edited the tables to reduce duplicate information

Screenshot 2020-10-26 at 5 43 07 PM

Screenshot 2020-10-26 at 5 34 18 PM

Screenshot 2020-10-26 at 5 34 07 PM

**Comment by PtrckM** [comment]
- Update current reward period
image
image

File: content/intro/liquidity-mining.mdx [hunk 1]
```diff
@@ -56,172 +56,36 @@ Work on removing this limit is in progress, you can read more about it in our [R
...
-      <td>RLC 180</td>
-    </tr>
-  </tbody>
-</table>
+**October 27, 2020 12:00am UTC - November 3, 2020 12:00am UTC**
+
+| Token                    | Trading Pair                            |  Exchange   | Maximum Sread | Spread Factor\* |     Weekly rewards     |
+| ------------------------ | :-------------------------------------- | :---------: | :-----------: | :-------------: | :--------------------: |
+| [COTI](#coti)            | COTI/BTC <br/> COTI/USDT <br/> COTI/BNB | Binance.com |      2%       |        8        |        USDC 250        |
+| [Mainframe](/#mainframe) | MFT/USDT <br/> MFT/ETH <br/> MFT/BNB    | Binance.com |      2%       |        8        |        USDC 250        |
```
Comment (dennisocana) [inline content/intro/liquidity-mining.mdx:74]: We no longer issue USDC rewards, change to USDT

**Review by dennisocana — CHANGES_REQUESTED** [review]

File: content/intro/liquidity-mining.mdx (hunk 1 above)
Comment (PtrckM) [reply content/intro/liquidity-mining.mdx:74]: image

**Pull request merged**